SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/CrossServerRPCService.cpp

```diff
@@ -398,6 +398,31 @@ void CrossServerRPCService::UpdateSentRPCsACKs(Worker_EntityId SenderId, const C
 			CrossServer::SentRPCEntry* SentRPC = SenderState.Mailbox.Find(RPCKey);
 			if (SentRPC != nullptr)
 			{
+				// If the ACK result is 'TargetUnknown' then resend the RPC immediately,
+				// as a temporary measure to bypass race conditions. There is deliberately
+				// no time-out handling at present.
+				if (ACK.Result == static_cast<uint64>(CrossServer::Result::TargetUnknown))
+				{
+					const Worker_EntityId TargetEntityId = SentRPC->Target.Entity;
+
+					const EntityViewElement& Entity = ActorSubView.GetView()[SenderId];
```

Comment:
Lots of use without checking, but since we are processing RPCs on the entity we are querying the maps, this is a sky-falling scenario, so this is reasonable assumption and checks would not improve the situation much.

Reply:
Thanks, I'll take a look, if it is going to fail I'd prefer to make it explicit.